### DISCUSSION TITLE
Multi currency

### DISCUSSION THREAD

**Discussion by Zamana** [EA31337/EA31337#336] [Ideas]
Hello!

I'm glad that this project exists.

Now, what I'm looking for, is an EA that can operate over a portfolio of stocks (or "multi currency" as people say).

Is this project "multi currency" ready? If not, would you consider to implement this feature?

Thanks.
Regards.

**Comment by kenorb** [comment]
Yes, the multi-currency trading feature is considered.
It can take few more releases till it's implemented.

**Reply by Zamana** [reply]
HI!

Thanks for replying.

I'm really looking for this, and I have some ideas regarding the portfolio management (balance control, quantity of different stocks, number of shares of each stock, sorting of stocks for decision, carrying position from one day to the next for Swing Trade etc).

Would be nice if I could participate.

**Reply by kenorb** [reply]
> I'm really looking for this, and I have some ideas regarding the portfolio management (balance control, quantity of different stocks, number of shares of each stock, sorting of stocks for decision, carrying position from one day to the next for Swing Trade etc). Would be nice if I could participate.

You can create a new discussion with a bullet point list (or multi discussions if different) with list of your ideas that you think would be worth implementing for EA, so we can come back to it at the convenient time.
For strategy ideas, check this [thread](https://github.com/EA31337/EA31337-strategies/discussions/categories/strategy-ideas). The different between ideas for EA and strategy, is that strategy should work based on specific and isolated method of trading (when to open or close orders), and EA algorithm shouldn't override strategy's logic, just help with the overall risk management.
